src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java

```diff
@@ -54,6 +55,30 @@ public static void main(String[] args) {
 		SoundEffect.BGM.play();
 	}
 
+	// Set up buttons
+	public void buttonOperation(JButton b, Color c){
```

Comment:
These should be private.  And can you rename `buttonOperation` to say what it does?  `setButtonStyle` or something?